packages/block-editor/README.md

```diff
@@ -569,6 +569,7 @@ _Properties_
 -   _\_\_experimentalBlockDirectory_ `boolean`: Whether the user has enabled the Block Directory
 -   _\_\_experimentalBlockPatterns_ `Array`: Array of objects representing the block patterns
 -   _\_\_experimentalBlockPatternCategories_ `Array`: Array of objects representing the block pattern categories
+-   _\_\_experimentalGenerateAnchors_ `boolean`: Enable/Disable auto anchor generation
```

Comment:
It would be good to explain the feature a little bit more:
```suggestion
-   _\_\_experimentalGenerateAnchors_ `boolean`: Enable/Disable auto anchor generation for Heading blocks
```